packages/wallet-sdk/src/core/communicator/Communicator.ts

```diff
@@ -62,24 +62,10 @@ export class Communicator {
     });
   };
 
-  /**
```

Comment:
we don't need to move this

Reply:
I moved it down since it's the only private method in the class now